### ISSUE TITLE
[Feature] Build MAL Debugger

### ISSUE THREAD

**Issue by wu-sheng** [apache/skywalking#7671]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

According to our recent added LAL debugger, we also should establish MAL debugger, which is a very similar tool, but focus on metrics analysis debug. The key of that, we should consider whether to accept multiple input format, and how users could get those rawdata from different tools, such as
1. Prometheus. It is the easiest, people could get the text from Prometheus endpoint fetch.
2. Native SkyWalking meter format, protobuf in JSON.
3. OC format, also protobuf in JSON

@wankai123 Let's talk about this in the 8.9.0 iteration and take a LAL debugger as a good reference.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Close this. I can see ppl are already easy to learn and use MAL directly. We are not considering providing this anymore.
If anyone wants to do this, we could discuss from there.

**Comment by anurnomeru** [comment]
But without a debugger, how can I observe the result to adjust my MAL rule?

**Comment by wu-sheng** [comment]
> But without a debugger, how can I observe the result to adjust my MAL rule?

This was closed due to MAL position changed since 9.x.
The MAL was planned for end users, which required too much dev experiences.
In v9, we began to ship official dashboards, MAL scripts and new UI configurations are out of box. All people begin to learn OAL, MAL and LAL should have basic coding capabilities. They are developer oriented now.
Many developers could understand the raw data and database structures now. So, this is not required.

Of course, this is a nice to have thing, as it could make development more friendly. But PMC team is not going to spend time on it anyway.

We are open to accept proposal and implementation if someone wants to add this.
